### ISSUE TITLE
ARM processes a disc up to "Starting Transcode With Handbrake" then acts as if it's finished but no media is ripped, no files created.

### ISSUE THREAD

**Issue by hometechblogger** [automatic-ripping-machine/automatic-ripping-machine#625]
- [x] I have checked the wiki
- [x] I have searched for similar issues

**Describe the bug**
On a fresh install of Ubuntu Server 20.04.5, on a physical PC, using the auto script installer and then applying the fix described in #587, ARM appears to function as it should but finishes too early and does not rip any media to any folders. Ripmethod is set to "backup" mode and mainfeature is set to "true". Log files are written.

In previous versions of ARM, I noticed that all blurays would be backed up to the raw folder, then transcoding would happen on the raw media. This does not happen now - the log file appears to show the system trying to transcode before anything is backed up to the raw folder. Not sure what has changed since the previous version(s) of ARM but I had it working for a least the last two years without issue.

**To Reproduce**
1. Insert bluray disc
2. Wait 5-10 minutes
3. Verifiy log file shows ARM processing complete but no media ripped / nothing transcoded.

### Environment
UbuntuServer 20.04.5
Latest version 2.6.0 stable from auto-install script.

### Log file

[JURASSIC_PARK_DOM.log](https://github.com/automatic-ripping-machine/automatic-ripping-machine/files/10149126/JURASSIC_PARK_DOM.log)

**Comment by hometechblogger** [comment]
Based on my very rudimentary debugging knowledge, this block below shows that directories are created in the folder structure for ARM and then the next phase is Handbrake trying to find the mainfeature. It identifies two possible tracks on the disc but then says they are under 600 seconds and drops them. 

So am I correct in assuming, in this particular case - handbrake can't identify the mainfeature track and thinks it's finished?

In the meantime I will test some other discs.

```
[12-04-2022 02:06:37] INFO ARM: main.log_arm_params ******************* End of config parameters *******************
[12-04-2022 02:06:37] INFO ARM: main.check_fstab Checking for fstab entry.
[12-04-2022 02:06:37] INFO ARM: main.check_fstab fstab entry is: /dev/sr0 /mnt/dev/sr0 udf,iso9660 user,noauto,exec,utf8 0 0
[12-04-2022 02:06:37] DEBUG ARM: utils.make_dir Creating directory: /home/arm/media/transcode/unidentified/Jurassic-Park (2011)
[12-04-2022 02:06:37] INFO ARM: utils.check_for_dupe_folder Final Output directory "/home/arm/media/transcode/unidentified/Jurassic-Park (2011)"
[12-04-2022 02:06:37] DEBUG ARM: utils.make_dir Creating directory: /home/arm/media/completed/unidentified/Jurassic-Park (2011)
[12-04-2022 02:06:37] INFO ARM: utils.check_for_dupe_folder Final Output directory "/home/arm/media/completed/unidentified/Jurassic-Park (2011)"
[12-04-2022 02:06:37] DEBUG ARM: utils.database_updater path=/home/arm/media/completed/unidentified/Jurassic-Park (2011):<class 'str'>
[12-04-2022 02:06:37] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 02:06:37] INFO ARM: arm_ripper.rip_visual_media Processing files to: /home/arm/media/transcode/unidentified/Jurassic-Park (2011)
[12-04-2022 02:06:37] DEBUG ARM: utils.database_updater status=transcoding:<class 'str'>
[12-04-2022 02:06:37] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 02:06:37] INFO ARM: arm_ripper.start_transcode ************* Starting Transcode With HandBrake *************
[12-04-2022 02:06:37] INFO ARM: utils.sleep_check_process Transcode limit is disabled
[12-04-2022 02:06:37] INFO ARM: handbrake.handbrake_all Starting BluRay/DVD transcoding - All titles
[12-04-2022 02:06:37] INFO ARM: handbrake.get_track_info Using HandBrake to get information on all the tracks on the disc.  This will take a few minutes...
[12-04-2022 02:06:37] DEBUG ARM: handbrake.get_track_info Sending command: HandBrakeCLI -i /dev/sr0 -t 0 --scan
[12-04-2022 02:11:00] DEBUG ARM: handbrake.get_track_info Line found is: [02:08:14] scan: BD has 69 title(s)
[12-04-2022 02:11:00] INFO ARM: handbrake.get_track_info Found 69 titles
[12-04-2022 02:11:00] DEBUG ARM: utils.put_track Track #55 Length:   59 fps: 23.976 aspect: 1.78 Mainfeature: False Source: HandBrake
[12-04-2022 02:11:00] DEBUG ARM: utils.database_adder Trying to add Track
[12-04-2022 02:11:00] DEBUG ARM: utils.database_adder successfully written Track to the database
[12-04-2022 02:11:00] DEBUG ARM: utils.put_track Track #59 Length:   23 fps: 23.976 aspect: 1.78 Mainfeature: False Source: HandBrake
[12-04-2022 02:11:00] DEBUG ARM: utils.database_adder Trying to add Track
[12-04-2022 02:11:00] DEBUG ARM: utils.database_adder successfully written Track to the database
[12-04-2022 02:11:00] DEBUG ARM: handbrake.handbrake_all Total number of tracks is 69
[12-04-2022 02:11:00] INFO ARM: handbrake.handbrake_all Track #55 of 69. Length (59) is less than minimum length (600). Skipping...
[12-04-2022 02:11:00] INFO ARM: handbrake.handbrake_all Track #59 of 69. Length (23) is less than minimum length (600). Skipping...
[12-04-2022 02:11:00] INFO ARM: handbrake.handbrake_all Handbrake processing complete
[12-04-2022 02:11:00] DEBUG ARM: handbrake.handbrake_all 

+--------------------+--------------------------------------------------------------+
|       Config       |                            Value                             |
+--------------------+--------------------------------------------------------------+
| _sa_instance_state |        <sqlalchemy.orm.state.InstanceState object at         |
|                    |                       0x7f6869987f70>                        |
|      logfile       |                    JURASSIC_PARK_DOM.log                     |
|     title_auto     |                        Jurassic-Park                         |
| video_type_manual  |                             None                             |
|      updated       |                            False                             |
|     mountpoint     |                         /mnt/dev/sr0                         |
|      imdb_id       |                             None                             |
|    title_manual    |                             None                             |
|    hasnicetitle    |                            False                             |
|     start_time     |                  2022-12-04 02:05:24.486551                  |
|        pid         |                             2673                             |
|     stop_time      |                             None                             |
|       job_id       |                              2                               |
|       errors       |                             None                             |
|      pid_hash      |                     -7403965808452075281                     |
|    imdb_id_auto    |                             None                             |
|        year        |                             2011                             |
|   imdb_id_manual   |                             None                             |
|     job_length     |                             None                             |
|     year_auto      |                             2011                             |
|      disctype      |                            bluray                            |
|    year_manual     |                             None                             |
|       status       |                         transcoding                          |
|       label        |                      JURASSIC_PARK_DOM                       |
|     poster_url     |                             None                             |
|        path        | /home/arm/media/completed/unidentified/Jurassic-Park (2011)  |
|       stage        |                         167011952421                         |
|     video_type     |                           unknown                            |
|  poster_url_auto   |                             None                             |
|    arm_version     |                            2.6.5                             |
|       crc_id       |                             None                             |
|      ejected       |                            False                             |
| poster_url_manual  |                             None                             |
|    no_of_titles    |                              69                              |
|  video_type_auto   |                             None                             |
|      devpath       |                           /dev/sr0                           |
|       title        |                        Jurassic-Park                         |
+--------------------+--------------------------------------------------------------+
[12-04-2022 02:11:00] DEBUG ARM: models.eject Unmounted disc /dev/sr0
[12-04-2022 02:11:07] DEBUG ARM: models.eject Failed to eject /dev/sr0
[12-04-2022 02:11:07] INFO ARM: arm_ripper.start_transcode ************* Finished Transcode With HandBrake *************
```

**Comment by 1337-server** [comment]
Blu-rays shouldn't be scanned with HandBrake. MakeMKV should always be used, not sure what failed here in the logic. Can you provide a full log ?

**Comment by hometechblogger** [comment]
Here's the full log from the OP:

```
[12-04-2022 04:27:40] DEBUG ARM: utils.check_db_version Head is: f1054468c1c7
[12-04-2022 04:27:40] DEBUG ARM: utils.check_db_version Database version is: f1054468c1c7
[12-04-2022 04:27:40] INFO ARM: utils.check_db_version Database is up to date
[12-04-2022 04:27:40] DEBUG ARM: utils.database_adder Trying to add User
[12-04-2022 04:27:40] DEBUG ARM: utils.database_adder successfully written User to the database
[12-04-2022 04:27:40] INFO ARM: main.<module> ************* Starting ARM processing at 2022-12-04 04:27:40.978218 *************
[12-04-2022 04:27:40] DEBUG ARM: utils.database_adder Trying to add Job
[12-04-2022 04:27:40] DEBUG ARM: utils.database_adder successfully written Job to the database
[12-04-2022 04:27:42] DEBUG ARM: utils.database_adder Trying to add Config
[12-04-2022 04:27:42] DEBUG ARM: utils.database_adder successfully written Config to the database
[12-04-2022 04:27:42] INFO ARM: main.<module> ARM version: 2.6.5
[12-04-2022 04:27:42] INFO ARM: main.<module> Python version: 3.8.10 (default, Jun 22 2022, 20:18:18) [GCC 9.4.0]
[12-04-2022 04:27:42] INFO ARM: main.<module> User is: arm
[12-04-2022 04:27:42] INFO ARM: logger.clean_up_logs Looking for log files older than 1 days old.
[12-04-2022 04:27:42] INFO ARM: main.<module> Job: JURASSIC_PARK_DOM
[12-04-2022 04:27:42] INFO ARM: utils.clean_old_jobs Job #1 with PID 2069 is currently running.
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ******************* Logging udev attributes *******************
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params DEVLINKS:/dev/disk/by-id/ata-ASUS_BW-16D1HT_KLLJ65E1811 /dev/disk/by-uuid/8ce1a705cff7c1a1 /dev/cdrw /dev/disk/by-path/pci-0000:02:00.1-ata-1 /dev/dvdrw /dev/dvd /dev/disk/by-label/JURASSIC_PARK_DOM /dev/cdrom
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params DEVNAME:/dev/sr0
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params DEVPATH:/devices/pci0000:00/0000:00:02.1/0000:02:00.1/ata1/host0/target0:0:0/0:0:0:0/block/sr0
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params DEVTYPE:disk
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_ATA:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_ATA_SATA:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_ATA_SATA_SIGNAL_RATE_GEN1:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_BUS:ata
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_BD:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_BD_R:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_BD_RE:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_CD:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_CD_R:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_CD_RW:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_PLUS_R:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_PLUS_RW:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_PLUS_R_DL:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_R:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_RAM:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_DVD_RW:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA_BD:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA_SESSION_COUNT:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA_STATE:complete
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA_TRACK_COUNT:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MEDIA_TRACK_COUNT_DATA:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MRW:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_CDROM_MRW_W:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FOR_SEAT:block-pci-0000_02_00_1-ata-1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_LABEL:JURASSIC_PARK_DOM
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_LABEL_ENC:JURASSIC_PARK_DOM
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_TYPE:udf
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_USAGE:filesystem
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_UUID:8ce1a705cff7c1a1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_UUID_ENC:8ce1a705cff7c1a1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_FS_VERSION:2.50
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_MODEL:ASUS_BW-16D1HT
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_MODEL_ENC:ASUS\x20\x20\x20\x20BW-16D1HT\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20\x20
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_PATH:pci-0000:02:00.1-ata-1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_PATH_TAG:pci-0000_02_00_1-ata-1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_REVISION:3.10
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_SCSI:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_SCSI_INQUIRY:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_SERIAL:ASUS_BW-16D1HT_KLLJ65E1811
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_SERIAL_SHORT:KLLJ65E1811
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_TYPE:cd
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_VENDOR:ASUS
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ID_VENDOR_ENC:ASUS\x20\x20\x20\x20
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params MAJOR:11
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params MINOR:0
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_MODEL:BW-16D1HT
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_MODEL_ENC:BW-16D1HT\x20\x20\x20\x20\x20\x20\x20
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_REVISION:3.10
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_TPGS:0
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_TYPE:cd/dvd
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_VENDOR:ASUS
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SCSI_VENDOR_ENC:ASUS\x20\x20\x20\x20
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SUBSYSTEM:block
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params SYSTEMD_MOUNT_DEVICE_BOUND:1
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params TAGS::seat:uaccess:systemd:
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params USEC_INITIALIZED:1875012
[12-04-2022 04:27:42] DEBUG ARM: main.log_udev_params ******************* End udev attributes *******************
[12-04-2022 04:27:42] INFO ARM: main.main Starting Disc identification
[12-04-2022 04:27:42] DEBUG ARM: identify.identify Identify Entry point --- job ----
[12-04-2022 04:27:42] INFO ARM: identify.identify Mounting disc to: /mnt/dev/sr0
[12-04-2022 04:27:46] DEBUG ARM: models.get_disc_type Found: /mnt/dev/sr0/BDMV
[12-04-2022 04:27:46] INFO ARM: identify.identify Disc identified as video
[12-04-2022 04:27:47] DEBUG ARM: identify.get_video_details Title = Jurassic-Park
[12-04-2022 04:27:47] DEBUG ARM: identify.get_video_details Title: Jurassic-Park | Year: 2011
[12-04-2022 04:27:47] DEBUG ARM: identify.get_video_details Calling webservice with title: Jurassic-Park and year: 2011
[12-04-2022 04:27:47] DEBUG ARM: identify.identify_loop Response = None
[12-04-2022 04:27:47] DEBUG ARM: identify.metadata_selector provider omdb
[12-04-2022 04:27:47] DEBUG ARM: metadata.call_omdb_api omdb call failed with error - HTTP Error 401: Unauthorized
[12-04-2022 04:27:47] DEBUG ARM: identify.try_with_year response: None
[12-04-2022 04:27:47] DEBUG ARM: identify.try_with_year Subtracting 1 year...
[12-04-2022 04:27:47] DEBUG ARM: identify.metadata_selector provider omdb
[12-04-2022 04:27:47] DEBUG ARM: metadata.call_omdb_api omdb call failed with error - HTTP Error 401: Unauthorized
[12-04-2022 04:27:47] DEBUG ARM: identify.try_without_year Removing year...
[12-04-2022 04:27:47] DEBUG ARM: identify.metadata_selector provider omdb
[12-04-2022 04:27:47] DEBUG ARM: metadata.call_omdb_api omdb call failed with error - HTTP Error 401: Unauthorized
[12-04-2022 04:27:47] DEBUG ARM: identify.try_without_year response: None
[12-04-2022 04:27:47] DEBUG ARM: identify.identify_loop Trying title: Jurassic
[12-04-2022 04:27:47] DEBUG ARM: identify.metadata_selector provider omdb
[12-04-2022 04:27:47] DEBUG ARM: metadata.call_omdb_api omdb call failed with error - HTTP Error 401: Unauthorized
[12-04-2022 04:27:47] DEBUG ARM: identify.identify_loop response: None
[12-04-2022 04:27:47] INFO ARM: identify.identify Disc title Post ident -  title:Jurassic-Park year:2011 video_type:unknown disctype: bluray
[12-04-2022 04:27:47] DEBUG ARM: identify.identify identify.job.end ---- 

+--------------------+--------------------------------------------------------------+
|       Config       |                            Value                             |
+--------------------+--------------------------------------------------------------+
| _sa_instance_state |        <sqlalchemy.orm.state.InstanceState object at         |
|                    |                       0x7fd01b268910>                        |
|      updated       |                            False                             |
|     mountpoint     |                         /mnt/dev/sr0                         |
|      logfile       |                    JURASSIC_PARK_DOM.log                     |
|     title_auto     |                        Jurassic-Park                         |
| video_type_manual  |                             None                             |
|    title_manual    |                             None                             |
|     start_time     |                  2022-12-04 04:27:40.978329                  |
|    hasnicetitle    |                            False                             |
|        pid         |                             2069                             |
|      imdb_id       |                             None                             |
|        year        |                             2011                             |
|      pid_hash      |                     -8317984020768307033                     |
|       errors       |                             None                             |
|     stop_time      |                             None                             |
|    imdb_id_auto    |                             None                             |
|       job_id       |                              1                               |
|     job_length     |                             None                             |
|     year_auto      |                             2011                             |
|   imdb_id_manual   |                             None                             |
|      disctype      |                            bluray                            |
|       status       |                            active                            |
|     poster_url     |                             None                             |
|       label        |                      JURASSIC_PARK_DOM                       |
|    year_manual     |                             None                             |
|       stage        |                         167012806072                         |
|        path        |                             None                             |
|     video_type     |                           unknown                            |
|  poster_url_auto   |                             None                             |
|    arm_version     |                            2.6.5                             |
|    no_of_titles    |                             None                             |
|      ejected       |                            False                             |
|       crc_id       |                             None                             |
|  video_type_auto   |                             None                             |
| poster_url_manual  |                             None                             |
|       title        |                        Jurassic-Park                         |
|      devpath       |                           /dev/sr0                           |
+--------------------+--------------------------------------------------------------+
[12-04-2022 04:27:47] DEBUG ARM: main.main Value of have_dupes: False
[12-04-2022 04:27:47] DEBUG ARM: utils.database_adder Trying to add Notifications
[12-04-2022 04:27:47] DEBUG ARM: utils.database_adder successfully written Notifications to the database
[12-04-2022 04:27:47] DEBUG ARM: utils.database_adder Trying to add Notifications
[12-04-2022 04:27:47] DEBUG ARM: utils.database_adder successfully written Notifications to the database
[12-04-2022 04:27:47] INFO ARM: utils.check_for_wait Waiting 60 seconds for manual override.
[12-04-2022 04:27:47] DEBUG ARM: utils.database_updater status=waiting:<class 'str'>
[12-04-2022 04:27:47] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater status=active:<class 'str'>
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params ******************* Logging ARM variables *******************
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params devpath: /dev/sr0
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params mountpoint: /mnt/dev/sr0
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params title: Jurassic-Park
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params year: 2011
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params video_type: unknown
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params hasnicetitle: False
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params label: JURASSIC_PARK_DOM
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params disctype: bluray
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params ******************* End of ARM variables *******************
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params ******************* Logging config parameters *******************
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params skip_transcode: False
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params mainfeature: True
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params minlength: 600
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params maxlength: 99999
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params videotype: auto
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params manual_wait: True
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params manual_wait_time: 60
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params ripmethod: backup
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params mkv_args: 
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params delrawfiles: True
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params hb_preset_dvd: HQ 720p30 Surround
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params hb_preset_bd: HQ 1080p30 Surround
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params hb_args_dvd: --subtitle scan -F
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params hb_args_bd: --subtitle scan -F --subtitle-burned --audio-lang-list eng --all-audio
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params raw_path: /home/arm/media/raw/
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params transcode_path: /home/arm/media/transcode/
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params completed_path: /home/arm/media/completed/
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params extras_sub: extras
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params emby_refresh: False
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params emby_server: 
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params emby_port: 8096
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params notify_rip: True
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params notify_transcode: True
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params max_concurrent_transcodes: 0
[12-04-2022 04:28:48] INFO ARM: main.log_arm_params ******************* End of config parameters *******************
[12-04-2022 04:28:48] INFO ARM: main.check_fstab Checking for fstab entry.
[12-04-2022 04:28:48] INFO ARM: main.check_fstab fstab entry is: /dev/sr0  /mnt/dev/sr0  udf,iso9660  users,noauto,exec,utf8  0  0
[12-04-2022 04:28:48] DEBUG ARM: utils.make_dir Creating directory: /home/arm/media/transcode/unidentified/Jurassic-Park (2011)
[12-04-2022 04:28:48] INFO ARM: utils.check_for_dupe_folder Final Output directory "/home/arm/media/transcode/unidentified/Jurassic-Park (2011)"
[12-04-2022 04:28:48] DEBUG ARM: utils.make_dir Creating directory: /home/arm/media/completed/unidentified/Jurassic-Park (2011)
[12-04-2022 04:28:48] INFO ARM: utils.check_for_dupe_folder Final Output directory "/home/arm/media/completed/unidentified/Jurassic-Park (2011)"
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater path=/home/arm/media/completed/unidentified/Jurassic-Park (2011):<class 'str'>
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 04:28:48] INFO ARM: arm_ripper.rip_visual_media Processing files to: /home/arm/media/transcode/unidentified/Jurassic-Park (2011)
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater status=transcoding:<class 'str'>
[12-04-2022 04:28:48] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 04:28:48] INFO ARM: arm_ripper.start_transcode ************* Starting Transcode With HandBrake *************
[12-04-2022 04:28:48] INFO ARM: utils.sleep_check_process Transcode limit is disabled
[12-04-2022 04:28:48] INFO ARM: handbrake.handbrake_all Starting BluRay/DVD transcoding - All titles
[12-04-2022 04:28:48] INFO ARM: handbrake.get_track_info Using HandBrake to get information on all the tracks on the disc.  This will take a few minutes...
[12-04-2022 04:28:48] DEBUG ARM: handbrake.get_track_info Sending command: HandBrakeCLI -i /dev/sr0 -t 0 --scan
[12-04-2022 04:33:13] DEBUG ARM: handbrake.get_track_info Line found is: [04:30:25] scan: BD has 69 title(s)
[12-04-2022 04:33:13] INFO ARM: handbrake.get_track_info Found 69 titles
[12-04-2022 04:33:13] DEBUG ARM: utils.put_track Track #55 Length:   59 fps: 23.976 aspect: 1.78 Mainfeature: False Source: HandBrake
[12-04-2022 04:33:13] DEBUG ARM: utils.database_adder Trying to add Track
[12-04-2022 04:33:13] DEBUG ARM: utils.database_adder successfully written Track to the database
[12-04-2022 04:33:13] DEBUG ARM: utils.put_track Track #59 Length:   23 fps: 23.976 aspect: 1.78 Mainfeature: False Source: HandBrake
[12-04-2022 04:33:13] DEBUG ARM: utils.database_adder Trying to add Track
[12-04-2022 04:33:13] DEBUG ARM: utils.database_adder successfully written Track to the database
[12-04-2022 04:33:13] DEBUG ARM: handbrake.handbrake_all Total number of tracks is 69
[12-04-2022 04:33:13] INFO ARM: handbrake.handbrake_all Track #55 of 69. Length (59) is less than minimum length (600). Skipping...
[12-04-2022 04:33:13] INFO ARM: handbrake.handbrake_all Track #59 of 69. Length (23) is less than minimum length (600). Skipping...
[12-04-2022 04:33:13] INFO ARM: handbrake.handbrake_all Handbrake processing complete
[12-04-2022 04:33:13] DEBUG ARM: handbrake.handbrake_all 

+--------------------+--------------------------------------------------------------+
|       Config       |                            Value                             |
+--------------------+--------------------------------------------------------------+
| _sa_instance_state |        <sqlalchemy.orm.state.InstanceState object at         |
|                    |                       0x7fd01b268910>                        |
|      updated       |                            False                             |
|     mountpoint     |                         /mnt/dev/sr0                         |
|      logfile       |                    JURASSIC_PARK_DOM.log                     |
|     title_auto     |                        Jurassic-Park                         |
| video_type_manual  |                             None                             |
|    title_manual    |                             None                             |
|     start_time     |                  2022-12-04 04:27:40.978329                  |
|    hasnicetitle    |                            False                             |
|        pid         |                             2069                             |
|      imdb_id       |                             None                             |
|        year        |                             2011                             |
|      pid_hash      |                     -8317984020768307033                     |
|       errors       |                             None                             |
|     stop_time      |                             None                             |
|    imdb_id_auto    |                             None                             |
|       job_id       |                              1                               |
|     job_length     |                             None                             |
|     year_auto      |                             2011                             |
|   imdb_id_manual   |                             None                             |
|      disctype      |                            bluray                            |
|       status       |                         transcoding                          |
|     poster_url     |                             None                             |
|       label        |                      JURASSIC_PARK_DOM                       |
|    year_manual     |                             None                             |
|       stage        |                         167012806072                         |
|        path        | /home/arm/media/completed/unidentified/Jurassic-Park (2011)  |
|     video_type     |                           unknown                            |
|  poster_url_auto   |                             None                             |
|    arm_version     |                            2.6.5                             |
|    no_of_titles    |                              69                              |
|      ejected       |                            False                             |
|       crc_id       |                             None                             |
|  video_type_auto   |                             None                             |
| poster_url_manual  |                             None                             |
|       title        |                        Jurassic-Park                         |
|      devpath       |                           /dev/sr0                           |
+--------------------+--------------------------------------------------------------+
[12-04-2022 04:33:13] DEBUG ARM: models.eject Unmounted disc /dev/sr0
[12-04-2022 04:33:19] DEBUG ARM: models.eject Failed to eject /dev/sr0
[12-04-2022 04:33:19] INFO ARM: arm_ripper.start_transcode ************* Finished Transcode With HandBrake *************
[12-04-2022 04:33:19] DEBUG ARM: utils.database_updater status=active:<class 'str'>
[12-04-2022 04:33:19] DEBUG ARM: utils.database_updater successfully written to the database
[12-04-2022 04:33:19] INFO ARM: utils.scan_emby EMBY_REFRESH config parameter is false.  Skipping emby scan.
[12-04-2022 04:33:19] INFO ARM: utils.delete_raw_files Removing raw path - /dev/sr0
[12-04-2022 04:33:21] DEBUG ARM: utils.delete_raw_files No raw files found to delete in /dev/sr0 - [Errno 123] No medium found: '/dev/sr0'
[12-04-2022 04:33:21] INFO ARM: utils.delete_raw_files Removing raw path - /home/arm/media/transcode/unidentified/Jurassic-Park (2011)
[12-04-2022 04:33:21] INFO ARM: utils.delete_raw_files Removing raw path - None
[12-04-2022 04:33:21] DEBUG ARM: utils.delete_raw_files No raw files found to delete in None - lstat: path should be string, bytes or os.PathLike, not NoneType
[12-04-2022 04:33:21] DEBUG ARM: utils.database_adder Trying to add Notifications
[12-04-2022 04:33:21] DEBUG ARM: utils.database_adder successfully written Notifications to the database
[12-04-2022 04:33:21] INFO ARM: arm_ripper.rip_visual_media ************* ARM processing complete *************

```

**Comment by ARMuser23** [comment]
I am also getting the same issue above, if arm is set to backup, blu-ray will go straigh to handbrake and skip the MakeMKV part. Test on 2.6.10 and 2.6.6

**Comment by ARMuser23** [comment]
any update on this issue? When i set 2.6.10 or 2.6.8 to backup mode, it also goes straight to transcoding with handbrake rather than ripping the content using makemkv into the raw folder.

**Comment by cmbernard333** [comment]
I am also seeing this issue myself. The docker container exhibits the exact same behavior. I tried using `mkv` mode but that appears to randomly select parts of the disc, rip them to raw, and transcode them. Previously with `backup` it would rip the entire disc contents and transcode all the elements.

**Comment by ARMuser23** [comment]
any update on this bug, this makes ARM unusable for me in the current state as it completely bypasses makemkv when mode is set to backup. Would it help providing any additional logs if so please advise which ones,

**Comment by 1337-server** [comment]
I've added a new PR that should fix this issue. If possible, help with testing fix would be great.

**Comment by 1337-server** [comment]
Thanks for testing. Hopefully we can get it merged soon to solve the problem for everyone.

**Comment by ARMuser23** [comment]
Could this fix be causing #728 ?
